.terraform-docs.yml

```diff
@@ -0,0 +1,15 @@
+formatter: "markdown table"
```

Comment:
do we need all this config? the default just has https://github.com/hmcts/terraform-module-template/blob/main/.terraform-docs.yml